assets/sass/authorize-application/_googlesitekit-authorize-application.scss

```
@@ -16,10 +16,138 @@
  * limitations under the License.
  */
 
-.auth-app-card #approve {
-	background-color: $c-deep-blue;
+.authorize-application-php {
+	background-color: $c-authorize-application-background;
+
+	.wrap {
+		margin: 30px 0 0;
+
+		h1 {
+			// This is the Authorize Application screen title.
+			font-size: $fs-authorize-application-headline-sm;
+			line-height: $lh-authorize-application-headline-sm;
+			padding: 0;
+		}
+	}
+
+	.auth-app-card {
+		border: none;
+		margin: 37px 0 0;
+		padding: 0;
+	}
+
+	.title {
+		font-size: $fs-authorize-application-title-lg;
+		font-weight: $fw-medium;
+		line-height: $lh-authorize-application-title-lg;
+		margin: 19px 0;
+	}
+
+	p {
+		font-size: $fs-authorize-application-label-lg;
+		font-weight: $fw-medium;
+		line-height: $lh-authorize-application-label-lg;
+		margin: 19px 0 20px;
+	}
+
+	strong {
+		font-weight: $fw-medium;
+	}
+
+	.form-field {
+		margin: 20px 0 40px;
+		max-width: 512px;
+
+		label {
```

We need to add the color `$c-authorize-application-sys-light-on-surface` (`#1f1f1f`) for the labels.
```suggestion
		label {
			color: $c-authorize-application-sys-light-on-surface;
```

---

Good spot, thanks!